src/System.Net.Primitives/src/System.Net.Primitives.csproj

```diff
@@ -51,8 +50,8 @@
     <Compile Include="System\Security\Authentication\ExtendedProtection\ChannelBinding.cs" />
     <Compile Include="System\Security\Authentication\ExtendedProtection\ChannelBindingKind.cs" />
 
-    <Compile Include="$(CommonPath)\Microsoft\Win32\SafeHandles\SafeLocalAllocHandle.cs" >
-      <Link>Common\Microsoft\Win32\SafeHandles\SafeLocalAllocHandle.cs</Link>
+    <Compile Include="$(CommonPath)\System\Net\ByteOrder.cs" >
```

Comment:
@pgavlin I can't find usage of ByteOrder within the Windows implementation. We should maybe compile it only for the *nix variants.